newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -80,6 +86,72 @@ export default class LocalProjectWriter {
     }
   };
 
+  static saveProjectAs = (project: gdProject): Promise<void> => {
+    const fileSystem = assignIn(new gd.AbstractFileSystemJS(), localFileSystem);
+    let filepath = project.getProjectFile();
+    const projectPath = path.dirname(project.getProjectFile());
+    const browserWindow = electron.remote.getCurrentWindow();
+    const options = {
+      defaultPath: projectPath + '\\game.json',
+      filters: [{ name: 'GDevelop 5 project', extensions: ['json'] }],
+    };
+
+    const newFilepath = dialog.showSaveDialog(browserWindow, options);
+    const newProjectFolder = path.dirname(newFilepath);
+
+    if (!filepath || filepath === '') {
+      return Promise.reject('Filepath from dialog is empty');
+    }
+
+    if (!dialog) {
+      return Promise.reject('Unimplemented dialog electron feature');
+    }
+
+    const serializedProjectObject = serializeToJSObject(project);
+    gd.ProjectResourcesCopier.copyAllResourcesTo(project, fileSystem, newProjectFolder, true, false, true);
+
+    if (project.isFolderProject()) {
```

Comment:
As this, until the end of the function, seems the exact copy of what is in saveProject, let's put this in a function `writeProjectFiles` (you can declare it after `writeJSONFile`) and call it:  `return writeProjectFiles(project, newFilepath)`. (maybe other arguments if needed)

Also include the line `const serializedProjectObject = serializeToJSObject(project);` inside, this should work